src/style.css

```diff
@@ -2,121 +2,209 @@ body {
   display: flex;
   align-items: center;
   justify-content: center;
-  margin: 0;
-  padding: 0;
-  box-sizing: border-box;
-  top: 200px;
-  position: relative;
-  background: aqua;
-  font-family: Verdana, Geneva, Tahoma, sans-serif;
+  height: 100vh;
+  width: 100%;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
 }
 
-.display {
+.todo-list {
+  width: 50%;
+  border: 1px solid #fff;
+  background-color: #fff;
+}
+
+.title {
+  border-bottom: 1px solid rgb(196, 196, 196);
+  width: 100%;
+  height: 45px;
   display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.title p {
+  padding-left: 10px;
+  font-size: 20px;
+  margin: 1px;
 }
 
-ul {
-  margin: 0;
-  padding: 0;
+.input-section {
+  height: 35px;
+  width: 100%;
+  border-bottom: 1px solid rgb(196, 196, 196);
+  line-height: 35px;
+}
+
+.todo-input {
+  width: 90%;
+  height: 90%;
+  border: transparent;
+  padding-left: 10px;
+  font-style: italic;
 }
 
-ul > * {
+.lists {
+  border-bottom: 1px solid rgb(196, 196, 196);
   list-style: none;
+  padding-left: 10px;
+  padding-right: 10px;
 }
 
-.container {
-  width: 60rem;
-  box-shadow: 5px 5px 11px 1px rgba(151, 151, 151, 0.63);
-  background: mistyrose;
+.list {
+  display: flex;
+  justify-content: space-between;
+  margin-bottom: 15px;
 }
 
-i::before {
-  font-size: 1.5rem;
-  color: rgba(146, 146, 146, 0.664);
+.list:hover {
+  background-color: mistyrose;
 }
 
-.container > *,
-.task-list > * {
-  border-bottom: 1px solid rgba(228, 228, 228, 0.664);
+.list:hover .desc {
+  background-color: mistyrose;
 }
 
-#form {
-  padding: 1.5rem 4%;
-  justify-content: space-between;
+.list p {
+  margin: 1px;
 }
 
-#input {
-  background: transparent;
-  border: none;
+.li-div {
+  display: flex;
 }
 
-::placeholder {
-  font-style: italic;
-  font-size: 1.3rem;
+#btn {
+  background-color: lightgoldenrodyellow;
+  border: transparent;
+  height: 30px;
+  width: 100%;
 }
 
-.header {
-  padding: 1.5rem 4%;
-  justify-content: space-between;
-  align-items: center;
+.draggable {
+  cursor: move;
 }
 
-.header h1 {
-  margin: 0;
+.line {
+  text-decoration: line-through;
 }
```

Comment:
- [ ] [OPTIONAL] Please make sure that this style persists after the page refresh for completed tasks.

Reply:
Alright, thank you.
Will try implement that.
Cheers!